operations/iuf/workflows/preparation.md

```diff
@@ -0,0 +1,48 @@
+# Prepare for the install or upgrade
+
+This section defines environment variables and directory content that is used throughout the workflow.
+
+**`NOTE`** The following step uses the `iuf activity` command to demonstrate how to record operations within an IUF activity. While `iuf` automatically records all `iuf run` operations within an IUF activity, any other
+administrative operation can also be recorded within an IUF activity by using `iuf activity` in this manner. The remainder of the workflow will not use `iuf activity`, deferring to the administrator to use it as they desire.
+`iuf activity` does not **need** to be used in this step, but the rest of the operations in the step are required.
+
+- [1. Prepare for the install or upgrade](#1-prepare-for-the-install-or-upgrade)
+- [2. Next steps](#2-next-steps)
+
+## 1. Prepare for the install or upgrade
+
+1. Create timestamped media, activity, and administrator directories on `ncn-m001`. Copy all distribution files from the HPC CSM Software Recipe to the media directory, utilizing `iuf activity` to record the time spent downloading
+media and associate it with activity `${ACTIVITY_NAME}`.
+
+    The following environment variables are used throughout the workflow:
+
+    | Name               | Path                                           | Description                                                                               |
+    | ------------------ | ---------------------------------------------- | ----------------------------------------------------------------------------------------- |
+    | `${ACTIVITY_NAME}` | n/a                                            | String identifier for the IUF activity and the `iuf -a` argument for all `iuf` commands   |
```

Comment:
Hmm.  The full paths are there because they are used throughout the documentation and also because they reside on the RBD mount.  I'm a little hesitant to drop them, but if you feel strongly, let's chat.

The next commit will have the ```Use a short...``` text you recommended.